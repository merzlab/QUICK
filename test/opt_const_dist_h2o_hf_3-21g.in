HF BASIS=3-21G CONSTRAIN cutoff=1.0e-9 denserms=1.0e-6 OPTIMIZE DIPOLE

O  0.0 0.0 0.0
H  1.0 0.0 0.0
H  0.0 1.0 0.0

DISTANCE 1 2
