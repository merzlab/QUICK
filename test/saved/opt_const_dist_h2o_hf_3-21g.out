| *********************************************************************************************************
| **                                                                                                     **
| **           888888888888                                                                              **
| **         8888888888888888                                                                            **
| **      888888888888888888                                                                             **
| **     888888888888888888                                                                              **
| **    888888888888888                                                                                  **
| **   88888888888888888888                               88888                       8888:              **
| **   8888888888888888888888Z                            88888                       8888:              **
| **   888888888888888888888888?                          88888                       8888:              **
| **   8888888888888      8888888                                        888888       8888:              **
| **    88888888888         888888     8888:     88888    88888        888888888I     8888:    888888    **
| **    8888888888           88888:    8888:     88888    88888      $888888888888    8888:   888888     **
| **    I8Z 88888             88888    8888:     88888    88888    .888888     8888   8888: 888888       **
| **    .8Z 88888             88888    8888:     88888    88888    $88888             8888:88888         **
| **     8I 88888      .=88. .88888    8888:     88888    88888    88888              8888888888         **
| **    :8  88888      888888$8888$    8888:     88888    88888    8888O              88888888888        **
| **   ,7   +88888.     8888888888.    8888:     88888    88888    888888             88888O888888       **
| **         $888888:.   .8888888      88888....888888    88888     888888     8888   8888:  888888      **
| **          I8888888888888888888     888888888888888    88888     O888888888888O    8888:   888888     **
| **            O888888888888888888     88888888888888    88888       88888888888$    8888:    888888    **
| **               8888888Z     888      .8888I  88888    88888         8888888       8888:     888888   **
| **                                                                                                     **
| **                                                                                                     **
| **                                         Copyright (c) 2025                                          **
| **                          Regents of the University of California San Diego                          **
| **                                    & Michigan State University                                      **
| **                                        All Rights Reserved.                                         **
| **                                                                                                     **
| **             This software is subject to the terms of the Mozilla Public License, v. 2.0.            **
| **                                                                                                     **
| *********************************************************************************************************
|                                                                                                          
| Cite this work as:                                                                                       
| Manathunga, M.; O'Hearn, K. A.; Shajan, A.; Smith, J.; Miao, Y.; He, X.; Ayers, K.;                      
| Brothers, E.; Palos, E.; Tripathy, V.; Goetz, A.W.; Merz, K.M. QUICK-25.03.                              
| University of California,San Diego, CA and                                                               
| Michigan State University, East Lansing, MI, 2025                                                        
|                                                                                                          
| If you have any comments or queries, please reach out to us on GitHub issues or discussion sections:     
| https://github.com/merzlab/QUICK.                                                                        
|                                                                                                          
| TASK STARTS ON: Thu Oct 30 10:31:32 2025
| INPUT FILE :    opt_const_dist_h2o_hf_3-21g.in
| OUTPUT FILE:    opt_const_dist_h2o_hf_3-21g.out
| BASIS SET PATH: /Users/andi/projects/quick/QUICK/install-dlf-constraints/basis

 @ Read Job And Atom

  KEYWORD=HF BASIS=3-21G CONSTRAIN CUTOFF=1.0E-9 DENSERMS=1.0E-6 OPTIMIZE DIPOLE
 ============== JOB CARD =============
 METHOD = HARTREE FOCK
 DIRECT SCF 
 SAD INITAL GUESS 
 USE DIIS SCF
 PRINT ENERGY EVERY CYCLE
 READ SAD GUESS FROM FILE
 DIPOLE
 GEOMETRY OPTIMIZATION ANALYTICAL GRADIENT
 REQUEST CRITERIA FOR GEOMETRY CONVERGENCE: 
      MAX ALLOWED GEO CHANGE  =  0.189E+00
      MAX GEOMETRY CHANGE     =  0.180E-02
      GEOMETRY CHANGE RMS     =  0.120E-02
      MAX GRADIENT CHANGE     =  0.100E-02
      GRADIENT NORMALIZATION  =  0.300E-03
      MAX OPTIMIZATION CYCLE  =     0
 GRADIENT CALCULATION
 MAX SCF CYCLES =    200
 MAX DIIS CYCLES =   10
 DELTA DENSITY START CYCLE =    3
 COMPUTATIONAL CUTOFF: 
      TWO-e INTEGRAL   =  0.100E-08
      BASIS SET PRIME  =  0.100E-09
      MATRIX ELEMENTS  =  0.100E-09
      BASIS FUNCTION   =  0.100E-05
      GRADIENT CUTOFF  =  0.100E-06
 DENSITY MATRIX MAXIMUM RMS FOR CONVERGENCE  =  0.100E-05
 BASIS SET = 3-21G,  TYPE = CARTESIAN
| BASIS FILE = /Users/andi/projects/quick/QUICK/install-dlf-constraints/basis/3-21G.BAS
| DATA FILE  =    opt_const_dist_h2o_hf_3-21g.dat

 @ Finish reading job


 @ Begin SAD initial guess

 For Atom Kind =    1
 ELEMENT = O 
 BASIS FUNCTIONS =    9
 For Atom Kind =    2
 ELEMENT = H 
 BASIS FUNCTIONS =    2

 @ Finish SAD initial guess


 @ Begin Reading Molecular Information


 =========== Molecule Input ==========
 TOTAL MOLECULAR CHARGE  =    0    MULTIPLICITY                =    1
 TOTAL ATOM NUMBER       =    3    NUMBER OF ATOM TYPES        =    2
 NUMBER OF HYDROGEN ATOM =    2    NUMBER OF NON-HYDROGEN ATOM =    1
 NUMBER OF ELECTRONS     =   10

 -- INPUT GEOMETRY -- :
    O           0.0000       0.0000       0.0000
    H           1.0000       0.0000       0.0000
    H           0.0000       1.0000       0.0000

 -- DISTANCE MATRIX -- :
              1           2           3
      1     0.00000
      2     1.88973     0.00000
      3     1.88973     2.67248     0.00000

============== BASIS INFOS ==============
 BASIS FUNCTIONS =   13
 NSHELL =    7 NPRIM  =   21
 JSHELL =    7 JBASIS =   12


 @ End Reading Molecular Information 


***********************************************************************
**                                                                   **
**                       ---------------------                       **
**                              DL-FIND                              **
**                       Geometry Optimisation                       **
**                       ---------------------                       **
**                                                                   **
**                 J. Kaestner, J.M. Carr, T.W. Keal,                **
**                W. Thiel, A. Wander and P. Sherwood                **
**                                                                   **
**              J. Phys. Chem. A, 2009, 113 (43), 11856.             **
**                                                                   **
**   Please include this reference in published work using DL-FIND.  **
**                                                                   **
**               Copyright:  STFC Daresbury Laboratory               **
**                           Revision:                               **
**                                                                   **
***********************************************************************

Residue member list:
    0 :     1    1    1

Generating new HDLC for    1 residues

Located a new residue for HDLC
   Residue number is    1
   Number of atoms it contains     3
   Atoms:      1     2     3
   Found constraint between atoms       1      2
   Generating HDLC for residue    1
     Considering constraint (type 1):     1    2
       ... constraining bond    1

   The Cartesian coordinates
         1     0.00000000000000     0.00000000000000     0.00000000000000
         2     1.88972613288564     0.00000000000000     0.00000000000000
         3     0.00000000000000     1.88972613288564     0.00000000000000

   The primitive internal coordinates


     The system has     2 stretches
         1 Stre     2     1              =    1.88972613
         2 Stre     3     1              =    1.88972613


     The system has     1 bends and impropers
         3 Bend     3     1     2        =   90.00000000 deg =    1.57079633


     The system has     0 dihedrals


   Residue      1 has internal coords      1 to      8

DL-FIND Report:
===============
Optimisation algorithm: L-BFGS
Number of steps in L-BFGS memory ...............        100

Step length: simple scaled
Maximum step length ............................  1.000E-01
Scaling step by ................................  1.000E+00

Coordinate system: Delocalised internal coordinates (DLC)

Number of atoms ................................          3
Number of input geometries .....................          1
Variables to be optimised ......................          2
No restart information is written
This run has not been restarted.
Maximum number of steps ........................         27
Maximum number of energy evaluations ...........     100000


 @ Optimize for New Step


============  GEOMETRY FOR OPTIMIZATION STEP   1  ============

GEOMETRY INPUT
ELEMENT      X              Y              Z
  O           0.000000       0.000000       0.000000
  H           1.000000       0.000000       0.000000
  H           0.000000       1.000000       0.000000

 @ Begin Energy Calculation

                                         SCF ENERGY
| ------------------------------------------------------------------------------------------------------------------------
| NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
| ------------------------------------------------------------------------------------------------------------------------
|  1    -75.501084278      ------         0.000     1        0.00      0.00      0.00    0.1518E+01  0.9668E-01  0.4101E+00
|  2    -75.548179622  0.470953E-01       0.000     2        0.00      0.00      0.00    0.1540E+00  0.2318E-01  0.1253E+00
|  3    -75.570534964  0.223553E-01       0.000     3        0.00      0.00      0.00    0.7143E-01  0.7470E-02  0.3344E-01
|  4    -75.574872623  0.433766E-02       0.001     4        0.00      0.00      0.00    0.1317E-01  0.3717E-02  0.1695E-01
|  5    -75.575412703  0.540080E-03       0.001     5        0.00      0.00      0.00    0.5331E-02  0.2441E-02  0.1153E-01
|  6    -75.575516089  0.103386E-03       0.001     6        0.00      0.00      0.00    0.7185E-03  0.3549E-03  0.1802E-02
|  7    -75.575518102  0.201315E-05       0.001     7        0.00      0.00      0.00    0.1129E-03  0.5596E-04  0.2664E-03
|  8    -75.575518166  0.642718E-07       0.001     8        0.00      0.00      0.00    0.1926E-04  0.1094E-04  0.3431E-04
|  9    -75.575518168  0.241871E-08       0.001     9        0.00      0.00      0.00    0.4562E-05  0.1952E-05  0.9270E-05
| 10    -75.575518169  0.641993E-10       0.001    10        0.00      0.00      0.00    0.4449E-06  0.1851E-06  0.7889E-06
| ------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER  10 CYCLES
| MAX ERROR = 0.444935E-06   RMS CHANGE = 0.185096E-06   MAX CHANGE = 0.788887E-06
| -----------------------------------------------
 ELECTRONIC ENERGY    =    -84.416538299
 CORE_CORE REPULSION  =      8.841020130
 TOTAL ENERGY         =    -75.575518169

 @ End Energy calculation


 ANALYTICAL GRADIENT: 
------------------------
 VARIBLES    NEW_GRAD
------------------------
    1X    -0.0490522409
    1Y    -0.0490522901
    1Z     0.0000001059
    2X     0.0160311555
    2Y     0.0330210690
    2Z    -0.0000000683
    3X     0.0330210854
    3Y     0.0160312211
    3Z    -0.0000000376
------------------------
 OPTIMIZATION STATISTICS:
 MAXIMUM GRADIENT ELEMENT=     0.5171303340E-01 (REQUEST=  0.10000E-02 )
 GRADIENT NORM           =     0.4555688107E-01 (REQUEST=  0.66667E-03 )

 @ Finish Optimization for This Step

 @ Optimize for New Step


============  GEOMETRY FOR OPTIMIZATION STEP   2  ============

GEOMETRY INPUT
ELEMENT      X              Y              Z
  O           0.003024       0.004334       0.000000
  H           1.002971      -0.005994       0.000000
  H          -0.005994       1.001660       0.000000

 @ Begin Energy Calculation

                                         SCF ENERGY
| ------------------------------------------------------------------------------------------------------------------------
| NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
| ------------------------------------------------------------------------------------------------------------------------
|  1    -75.579979480      ------         0.001     1        0.00      0.00      0.00    0.3872E-02  0.8365E-03  0.2822E-02
|  2    -75.576759169  -.322031E-02       0.000     2        0.00      0.00      0.00    0.9372E-03  0.2129E-03  0.7550E-03
|  3    -75.576761414  0.224446E-05       0.000     3        0.00      0.00      0.00    0.3469E-03  0.1159E-03  0.4806E-03
|  4    -75.576761827  0.412936E-06       0.000     4        0.00      0.00      0.00    0.8758E-04  0.3127E-04  0.1692E-03
|  5    -75.576761857  0.301496E-07       0.000     5        0.00      0.00      0.00    0.3085E-04  0.1493E-04  0.5733E-04
|  6    -75.576761861  0.454171E-08       0.000     6        0.00      0.00      0.00    0.5315E-05  0.1513E-05  0.5554E-05
|  7    -75.576761861  0.576872E-10       0.000     7        0.00      0.00      0.00    0.9799E-06  0.2021E-06  0.7930E-06
| ------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER   7 CYCLES
| MAX ERROR = 0.979870E-06   RMS CHANGE = 0.202079E-06   MAX CHANGE = 0.793040E-06
| -----------------------------------------------
 ELECTRONIC ENERGY    =    -84.425876224
 CORE_CORE REPULSION  =      8.849114362
 TOTAL ENERGY         =    -75.576761861

 @ End Energy calculation


 ANALYTICAL GRADIENT: 
------------------------
 VARIBLES    NEW_GRAD
------------------------
    1X    -0.0480000223
    1Y    -0.0456779000
    1Z     0.0000000002
    2X     0.0173223475
    2Y     0.0305547459
    2Z    -0.0000000001
    3X     0.0306776748
    3Y     0.0151231541
    3Z    -0.0000000001
------------------------
 OPTIMIZATION STATISTICS:
 ENERGY CHANGE           =     0.1243692878E-02 (REQUEST=  0.10000E-05 )
 MAXIMUM GEOMETRY CHANGE =     0.1605317824E-01 (REQUEST=  0.40000E-02 )
 GEOMETRY CHANGE RMS     =     0.1414213562E-01 (REQUEST=  0.26667E-02 )
 MAXIMUM GRADIENT ELEMENT=     0.4807011324E-01 (REQUEST=  0.10000E-02 )
 GRADIENT NORM           =     0.4238571159E-01 (REQUEST=  0.66667E-03 )
Scaling step back by    0.47202

 @ Finish Optimization for This Step

 @ Optimize for New Step


============  GEOMETRY FOR OPTIMIZATION STEP   3  ============

GEOMETRY INPUT
ELEMENT      X              Y              Z
  O           0.022944       0.031713       0.000000
  H           1.020144      -0.043073       0.000000
  H          -0.043089       1.011360       0.000000

 @ Begin Energy Calculation

                                         SCF ENERGY
| ------------------------------------------------------------------------------------------------------------------------
| NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
| ------------------------------------------------------------------------------------------------------------------------
|  1    -75.601695061      ------         0.000     1        0.00      0.00      0.00    0.2390E-01  0.4990E-02  0.1709E-01
|  2    -75.582458737  -.192363E-01       0.000     2        0.00      0.00      0.00    0.5457E-02  0.1276E-02  0.4549E-02
|  3    -75.582538035  0.792986E-04       0.000     3        0.00      0.00      0.00    0.1964E-02  0.7379E-03  0.3492E-02
|  4    -75.582552802  0.147671E-04       0.000     4        0.00      0.00      0.00    0.4877E-03  0.2359E-03  0.1015E-02
|  5    -75.582553894  0.109133E-05       0.000     5        0.00      0.00      0.00    0.1452E-03  0.3069E-04  0.1631E-03
|  6    -75.582553924  0.306621E-07       0.000     6        0.00      0.00      0.00    0.2285E-04  0.8749E-05  0.2796E-04
|  7    -75.582553926  0.173221E-08       0.000     7        0.00      0.00      0.00    0.5867E-05  0.1105E-05  0.4523E-05
|  8    -75.582553926  0.453628E-10       0.000     8        0.00      0.00      0.00    0.4240E-06  0.9087E-07  0.2967E-06
| ------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER   8 CYCLES
| MAX ERROR = 0.424041E-06   RMS CHANGE = 0.908740E-07   MAX CHANGE = 0.296742E-06
| -----------------------------------------------
 ELECTRONIC ENERGY    =    -84.480950206
 CORE_CORE REPULSION  =      8.898396280
 TOTAL ENERGY         =    -75.582553926

 @ End Energy calculation


 ANALYTICAL GRADIENT: 
------------------------
 VARIBLES    NEW_GRAD
------------------------
    1X    -0.0404434475
    1Y    -0.0229823009
    1Z     0.0000000000
    2X     0.0241658783
    2Y     0.0147254573
    2Z    -0.0000000000
    3X     0.0162775693
    3Y     0.0082568436
    3Z    -0.0000000000
------------------------
 OPTIMIZATION STATISTICS:
 ENERGY CHANGE           =     0.5792064609E-02 (REQUEST=  0.10000E-05 )
 MAXIMUM GEOMETRY CHANGE =     0.1000000000E+00 (REQUEST=  0.40000E-02 )
 GEOMETRY CHANGE RMS     =     0.8925930024E-01 (REQUEST=  0.26667E-02 )
 MAXIMUM GRADIENT ELEMENT=     0.2515584858E-01 (REQUEST=  0.10000E-02 )
 GRADIENT NORM           =     0.2260805559E-01 (REQUEST=  0.66667E-03 )
Scaling step back by    0.90983

 @ Finish Optimization for This Step

 @ Optimize for New Step


============  GEOMETRY FOR OPTIMIZATION STEP   4  ============

GEOMETRY INPUT
ELEMENT      X              Y              Z
  O           0.045431       0.059859       0.000000
  H           1.035495      -0.080760       0.000000
  H          -0.080926       1.020902       0.000000

 @ Begin Energy Calculation

                                         SCF ENERGY
| ------------------------------------------------------------------------------------------------------------------------
| NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
| ------------------------------------------------------------------------------------------------------------------------
|  1    -75.600348166      ------         0.000     1        0.00      0.00      0.00    0.2115E-01  0.4795E-02  0.1786E-01
|  2    -75.584840863  -.155073E-01       0.000     2        0.00      0.00      0.00    0.5078E-02  0.1215E-02  0.4399E-02
|  3    -75.584913628  0.727655E-04       0.000     3        0.00      0.00      0.00    0.1765E-02  0.6730E-03  0.3540E-02
|  4    -75.584927364  0.137355E-04       0.001     4        0.00      0.00      0.00    0.9846E-03  0.3122E-03  0.1287E-02
|  5    -75.584930035  0.267131E-05       0.001     5        0.00      0.00      0.00    0.1781E-03  0.9256E-04  0.4317E-03
|  6    -75.584930171  0.135830E-06       0.001     6        0.00      0.00      0.00    0.2029E-04  0.7694E-05  0.2535E-04
|  7    -75.584930172  0.133778E-08       0.001     7        0.00      0.00      0.00    0.4674E-05  0.8905E-06  0.3642E-05
| ------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER   7 CYCLES
| MAX ERROR = 0.467433E-05   RMS CHANGE = 0.890545E-06   MAX CHANGE = 0.364161E-05
| -----------------------------------------------
 ELECTRONIC ENERGY    =    -84.523170546
 CORE_CORE REPULSION  =      8.938240374
 TOTAL ENERGY         =    -75.584930172

 @ End Energy calculation


 ANALYTICAL GRADIENT: 
------------------------
 VARIBLES    NEW_GRAD
------------------------
    1X    -0.0312947065
    1Y     0.0007631086
    1Z     0.0000000000
    2X     0.0289688960
    2Y    -0.0017331118
    2Z     0.0000000000
    3X     0.0023258105
    3Y     0.0009700032
    3Z    -0.0000000000
------------------------
 OPTIMIZATION STATISTICS:
 ENERGY CHANGE           =     0.2376246037E-02 (REQUEST=  0.10000E-05 )
 MAXIMUM GEOMETRY CHANGE =     0.1000000000E+00 (REQUEST=  0.40000E-02 )
 GEOMETRY CHANGE RMS     =     0.9320415623E-01 (REQUEST=  0.26667E-02 )
 MAXIMUM GRADIENT ELEMENT=     0.3314438483E-02 (REQUEST=  0.10000E-02 )
 GRADIENT NORM           =     0.3184714301E-02 (REQUEST=  0.66667E-03 )

 @ Finish Optimization for This Step

 @ Optimize for New Step


============  GEOMETRY FOR OPTIMIZATION STEP   5  ============

GEOMETRY INPUT
ELEMENT      X              Y              Z
  O           0.049409       0.064244       0.000000
  H           1.037898      -0.087052       0.000000
  H          -0.087307       1.022808       0.000000

 @ Begin Energy Calculation

                                         SCF ENERGY
| ------------------------------------------------------------------------------------------------------------------------
| NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
| ------------------------------------------------------------------------------------------------------------------------
|  1    -75.586569007      ------         0.001     1        0.00      0.00      0.00    0.2757E-02  0.7532E-03  0.3089E-02
|  2    -75.584979288  -.158972E-02       0.001     2        0.00      0.00      0.00    0.7866E-03  0.1858E-03  0.6798E-03
|  3    -75.584981047  0.175928E-05       0.001     3        0.00      0.00      0.00    0.3000E-03  0.1009E-03  0.5503E-03
|  4    -75.584981404  0.356589E-06       0.000     4        0.00      0.00      0.00    0.1597E-03  0.6364E-04  0.2798E-03
|  5    -75.584981495  0.914444E-07       0.000     5        0.00      0.00      0.00    0.2520E-04  0.1280E-04  0.5565E-04
|  6    -75.584981498  0.254932E-08       0.000     6        0.00      0.00      0.00    0.2523E-05  0.9078E-06  0.2898E-05
| ------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER   6 CYCLES
| MAX ERROR = 0.252286E-05   RMS CHANGE = 0.907778E-06   MAX CHANGE = 0.289783E-05
| -----------------------------------------------
 ELECTRONIC ENERGY    =    -84.525390155
 CORE_CORE REPULSION  =      8.940408658
 TOTAL ENERGY         =    -75.584981498

 @ End Energy calculation


 ANALYTICAL GRADIENT: 
------------------------
 VARIBLES    NEW_GRAD
------------------------
    1X    -0.0295619377
    1Y     0.0039305883
    1Z     0.0000000000
    2X     0.0295337003
    2Y    -0.0044247069
    2Z    -0.0000000000
    3X     0.0000282374
    3Y     0.0004941186
    3Z    -0.0000000000
------------------------
 OPTIMIZATION STATISTICS:
 ENERGY CHANGE           =     0.5132556961E-04 (REQUEST=  0.10000E-05 )
 MAXIMUM GEOMETRY CHANGE =     0.1564789539E-01 (REQUEST=  0.40000E-02 )
 GEOMETRY CHANGE RMS     =     0.1543102356E-01 (REQUEST=  0.26667E-02 )
 MAXIMUM GRADIENT ELEMENT=     0.4895925719E-03 (REQUEST=  0.10000E-02 )
 GRADIENT NORM           =     0.3656024482E-03 (REQUEST=  0.66667E-03 )

 @ Finish Optimization for This Step

 @ Optimize for New Step


============  GEOMETRY FOR OPTIMIZATION STEP   6  ============

GEOMETRY INPUT
ELEMENT      X              Y              Z
  O           0.049465       0.064950       0.000000
  H           1.037834      -0.087123       0.000000
  H          -0.087299       1.022172       0.000000

 @ Begin Energy Calculation

                                         SCF ENERGY
| ------------------------------------------------------------------------------------------------------------------------
| NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
| ------------------------------------------------------------------------------------------------------------------------
|  1    -75.585770804      ------         0.000     1        0.00      0.00      0.00    0.2605E-02  0.1087E-03  0.4509E-03
|  2    -75.584981145  -.789659E-03       0.000     2        0.00      0.00      0.00    0.1952E-03  0.3574E-04  0.1341E-03
|  3    -75.584981211  0.664000E-07       0.000     3        0.00      0.00      0.00    0.4517E-04  0.1485E-04  0.6306E-04
|  4    -75.584981218  0.681800E-08       0.000     4        0.00      0.00      0.00    0.1601E-04  0.5357E-05  0.2093E-04
|  5    -75.584981219  0.824189E-09       0.000     5        0.00      0.00      0.00    0.6017E-05  0.2098E-05  0.1079E-04
|  6    -75.584981219  0.950866E-10       0.000     6        0.00      0.00      0.00    0.1326E-05  0.4767E-06  0.1966E-05
| ------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER   6 CYCLES
| MAX ERROR = 0.132592E-05   RMS CHANGE = 0.476745E-06   MAX CHANGE = 0.196595E-05
| -----------------------------------------------
 ELECTRONIC ENERGY    =    -84.531464799
 CORE_CORE REPULSION  =      8.946483580
 TOTAL ENERGY         =    -75.584981219

 @ End Energy calculation


 ANALYTICAL GRADIENT: 
------------------------
 VARIBLES    NEW_GRAD
------------------------
    1X    -0.0297406673
    1Y     0.0053128928
    1Z     0.0000000000
    2X     0.0295848579
    2Y    -0.0045119190
    2Z    -0.0000000000
    3X     0.0001558094
    3Y    -0.0008009738
    3Z    -0.0000000000
------------------------
 OPTIMIZATION STATISTICS:
 ENERGY CHANGE           =     0.2789463025E-06 (REQUEST=  0.10000E-05 )
 MAXIMUM GEOMETRY CHANGE =     0.2591953266E-02 (REQUEST=  0.40000E-02 )
 GEOMETRY CHANGE RMS     =     0.1911925053E-02 (REQUEST=  0.26667E-02 )
 MAXIMUM GRADIENT ELEMENT=     0.5864708509E-03 (REQUEST=  0.10000E-02 )
 GRADIENT NORM           =     0.5786861927E-03 (REQUEST=  0.66667E-03 )

 GEOMETRY OPTIMIZED AFTER    6 CYCLES

 @ Finish Optimization for This Step 

================ OPTIMIZED GEOMETRY INFORMATION ==============

 OPTIMIZED GEOMETRY IN CARTESIAN
 ELEMENT      X              Y              Z
  O           0.049465       0.064950       0.000000
  H           1.037834      -0.087123       0.000000
  H          -0.087299       1.022172       0.000000

 FORCE
 ELEMENT      X              Y              Z
  O           0.015738      -0.002811      -0.000000
  H          -0.015656       0.002388       0.000000
  H          -0.000082       0.000424       0.000000

 MINIMIZED ENERGY =    -75.584981219
===============================================================

 @ Finish Optimization Job


DL-FIND Report:
===============
Optimisation algorithm: L-BFGS
Number of steps in L-BFGS memory ...............        100

Step length: simple scaled
Maximum step length ............................  1.000E-01
Scaling step by ................................  1.000E+00

Coordinate system: Delocalised internal coordinates (DLC)

Number of atoms ................................          3
Number of input geometries .....................          1
Variables to be optimised ......................          2
No restart information is written
This run has not been restarted.
Number of energy evaluations on this processor .          6
Number of steps ................................          6
Number of accepted steps / line searches .......          6


 @ Begin Charge and Dipole Calculation

    ATOMIC CHARGES
   ATOM          MULLIKEN            LOWDIN
   O             -0.7242            -0.4703
   H              0.3577             0.2365
   H              0.3665             0.2338
   TOTAL         0.0000             0.0000

    DIPOLE (DEBYE)
      X         Y         Z        TOTAL
    1.7108    1.6846    0.0000    2.4010

 @ End Charge and Dipole Calculation


 @ Output Timing Information

------------- TIMING ---------------
| INITIAL GUESS TIME  =     0.081936000( 51.82%)
| TOTAL SCF TIME      =     0.022613000( 14.30%)
|       TOTAL OP TIME      =     0.021358000( 13.51%)
|             TOTAL 1e TIME      =     0.000550000(  0.35%)
|             TOTAL 2e TIME      =     0.019965000( 12.63%)
|       TOTAL DII TIME      =     0.001153000(  0.73%)
|             TOTAL DIAG TIME    =     0.000723000(  0.46%)
| TOTAL GRADIENT TIME      =     0.014466000(  9.15%)
|       TOTAL 1e GRADIENT TIME      =     0.001687000( 1.32%)
|       TOTAL 2e GRADIENT TIME      =     0.012358000( 7.82%)
| TOTAL TIME          =     0.158120000
------------------------------------
| Job cpu time:  0 days  0 hours  0 minutes  0.2 seconds.

 @ Finish Output Timing Information

| Normal Termination. Task Finished on: Thu Oct 30 10:31:32 2025
